src/main/resources/templates/admin/create.html

```diff
@@ -37,7 +37,7 @@ <h2>Current Admins</h2>
                   th:name="${_csrf.parameterName}"
                   th:value="${_csrf.token}"
                 />
-                <button  th:if="${!a.isPermanentAdmin}" type="submit" class="btn btn-danger">Delete</button>
+                <button  th:if="${!a.isPermanentAdmin}" type="submit" id="selenium-admin-delete" class="btn btn-danger">Delete</button>
```

Comment:
Is there any particular reason why the `id` named like this? `id` has an actual meaning in HTML and it's not unreasonable that we might need to use this id in a non-selenium context. Maybe we should replace this with something like `data-selenium="admin-delete"`?

Reply:
That's a good suggestion